### DISCUSSION TITLE
[FEATURE] opensumi支持Selenium自动化测试吗

### DISCUSSION THREAD

**Discussion by 767746649zyy** [opensumi/core#2159] [Q&A]
opensumi支持Selenium自动化测试吗，有具体的使用例子吗

**Comment by erha19** [comment]
@767746649zyy 目前 OpenSumi 内采用的是 Playwright 作为自动化测试框架，可以看一下这里 https://github.com/opensumi/core/discussions/2108#discussioncomment-4475707

**Reply by 767746649zyy** [reply]
我看到electron手册上支持selenium自动化测试，链接，不知道我可以在opensumi里也这么做吗？[示例链接](https://wizardforcel.gitbooks.io/electron-doc/content/tutorial/using-selenium-and-webdriver.html) 但是好像资源文件结构有不一样 设置连接ChromeDriver，以及 Electron 的路径的代码要写在哪里

**Reply by erha19** [reply]
@767746649zyy 理论上可行，但需要你自己去实现，Playwright 也有针对 Electron 的启动点写法 https://playwright.dev/docs/api/class-electron ，可以参考一下
